setup.py

```diff
@@ -1,27 +1,19 @@
-import subprocess
-import sys
 from setuptools import setup, find_packages
 
-try:
-    v = subprocess.run(['git', 'describe', '--tags'],
-                       stdout=subprocess.PIPE).stdout.strip().decode()
-    __version__ = v[: v.rfind("-")].replace("-", ".dev") if "-" in v else v
-except Exception as e:
-    raise RuntimeError("Could not get version from Git repo") from e
 
-
-package_info = dict(
+setup(
     name='neurodamus',
```

Comment:
This destroys the ability of setup.py being importable. It it sometimes useful for getting information